bundles/best_practices.rst

```diff
@@ -227,14 +227,15 @@ Require a Specific Symfony Version
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 You can use the special ``SYMFONY_REQUIRE`` environment variable together
-with Symfony Flex to install a specific Symfony version:
```

Comment:
Why this change?

Reply:
I agree. Reverted while merging. Thanks!